src/styles/blocks/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  color: $main-text-color;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-bottom: 4px;
+  }
+
+  &__product_code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+  }
```

Use scss nesting everywhere
```suggestion
  &__reviews {
    display: flex;
    justify-content: space-between;
    align-items: flex-end;
    margin-bottom: 24px;
    
    &-count {
      // write styles here
    }
  }

```